test/test_ops.py

```diff
@@ -1909,6 +1908,7 @@ def test_refs_are_in_decomp_table(self, op):
     skip("linalg.pinv", "hermitian"),
     skip("linalg.pinv", "singular"),
     skip('pinverse'),
+    skip('cholesky_inverse'),  # test/test_ops.py -k test_fake_crossref_backward_amp_cholesky_inverse_cuda_float32
```

Comment:
Would prefer if we didn't regress the tests to land this - could you look into this ?

Reply:
Fixed. It was flaky, and not failing now...